server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/materials/perforce_material_representer.rb

```diff
@@ -18,12 +18,16 @@ module ApiV1
   module Config
     module Materials
       class PerforceMaterialRepresenter < ScmMaterialRepresenter
+        P4MaterialConfig.send(:field_accessor, 'password')
```

Comment:
@jyotisingh can you review this PR, this line (and similar lines in other classes) feels like a really bad hack. But I don't know how to cleanly get away with something similar without resorting to breaking the old pages.